src/yggdrasil/api.go

```diff
@@ -363,6 +363,22 @@ func (c *Core) SetNodeInfo(nodeinfo interface{}, nodeinfoprivacy bool) {
 	c.router.nodeinfo.setNodeInfo(nodeinfo, nodeinfoprivacy)
 }
 
+// GetMaximumSessionMTU returns the maximum allowed session MTU size.
+func (c *Core) GetMaximumSessionMTU(mtu uint16) uint16 {
+	return c.router.sessions.myMaximumMTU
```

Comment:
This field can only be safely read/written by the router actor (since it needs to check it to create sessions), unless we can guarantee that `c.router.sessions.myMaximumMTU` remains constant after it is first set (and I don't think we can guarantee that).

So this and `SetMaximumSessionMTU` below both need to go through the actor. A `phony.Block` would probably be easiest.